godot_fmod.h

```diff
@@ -175,3 +197,7 @@ class Fmod : public Object {
 	Fmod();
 	~Fmod();
 };
+
+extern Mutex *mut;
```

Comment:
Best to wrap these definitions inside of a namespace. Leaving them out in the open may introduce problems later on.